Base.lproj/Front.strings

```diff
@@ -1,168 +1,168 @@
+"Should continue moving and overwrite exisiting file(s) ?" = "Should continue moving and overwrite exisiting file(s) ?";
+
+"Rename to \"{0}\" ?" = "Rename to \"{0}\" ?";
+"Failed to rename from \"{0}\" to \"{1}\"" = "Failed to rename from \"{0}\" to \"{1}\"";
+
+/* Image upload & move */
+"Successfully uploaded {0} images." = "Successfully uploaded {0} images.";
+"Failed to upload image. Error Message:" = "Failed to upload image. Error Message:";
+"Failed to Copy Image to Target Folder." = "Failed to Copy Image to Target Folder.";
+"Select Container Folder" = "Select Container Folder";
+
+"Saved" = "Saved";
+
+/* place holder in find & replace input */
+"search text" = "search text";
+"replace with" = "replace with";
+/* search option */
+"Case Sensitive" = "Case Sensitive";
+"Whole Word" = "Whole Word";
+
+"input image URL" = "input image URL";
+"Select image from local file" = "Select image from local file";
+"Cannot open location <code>$1</code>, do you mean <a href='https://$3'>https://$2</a> ?" = "Cannot open location <code>$1</code>, do you mean <a href='https://$3'>https://$2</a> ?";
+
+"select a language" = "select a language";
+"Finish Editing" = "Finish Editing";
+
+"Revert advanced setting file to default one ?" = "Revert advanced setting file to default one ?";
+
+"Select Spell Check Language" = "Select Spell Check Language";
+"Auto Detect Language" = "Auto Detect Language";
+"Disable Spell Check" = "Disable Spell Check";
+"Global Setting…" = "Global Setting…";
+"Dictionary file is missing to enable spellcheck support for %@." = "Dictionary file is missing to enable spellcheck support for %@.";
+"Install Language Support" = "Install Language Support";
+"License Information" = "License Information";
+"Download" = "Download";
+"Install dictionary file only for Typora" = "Install dictionary file only for Typora";
+"Spell Check (%@ unavailable)" = "Spell Check (%@ unavailable)";
+"Spell Check (%@ downloading)" = "Spell Check (%@ downloading)";
+"Download dictionary file for %@" = "Download dictionary file for %@";
+"(Default spell check option)" = "(Default spell check option)";
+"Failed to load spellcheck module" = "Failed to load spellcheck module";
+
+"<strong>%@</strong> has been turned on." = "<strong>%@</strong> has been turned on.";
+"You could turn it off from `View` menu." = "You could turn it off from `View` menu.";
+"The file is too large to render in Typora." = "The file is too large to render in Typora.";
+"Open in…" = "Open in…";
+"Quick Look" = "Quick Look";
+
+"Filter" = "Filter";
+"No result found." = "No result found.";
+"Continue & Overwrite" = "Continue & Overwrite";
+"The file is read-only, or you do not have right permissions to modify the file." = "The file is read-only, or you do not have right permissions to modify the file.";
+
+"Learn Data Recovery" = "Learn Data Recovery";
```

Comment:
Could you revert unchanged part of this file? It's hard to find out which part is changed